### ISSUE TITLE
Unhealthy instances after deployment leads to 502 Bad Gateway

### ISSUE THREAD

**Issue by thiagoscodelerae** [aws-ia/terraform-aws-polygon-technology-edge#18]
After deploying that module on AWS, the 4 EC2 instances are in "unhealthy" state on the load balancer target group:

![image](https://user-images.githubusercontent.com/110118896/209583415-3b0c770e-6140-44f8-ae70-90e3f7ed240b.png)

The deployment went successfully but it seems there is nothing listening at port 8545 on the instance:

`
$ sudo lsof -i -P -n | grep LISTEN
systemd-r  438 systemd-resolve   13u  IPv4  19831      0t0  TCP 127.0.0.53:53 (LISTEN)
sshd      1257            root    3u  IPv4  25137      0t0  TCP *:22 (LISTEN)
sshd      1257            root    4u  IPv6  25148      0t0  TCP *:22 (LISTEN)
`

Deployment done by following these instructions: https://wiki.polygon.technology/docs/edge/get-started/terraform-aws-deployment/

Response I got when running the curl command described on the documentation above:
`
502 Bad Gateway
`

**Comment by thiagoscodelerae** [comment]
This issue has been solved by fixing the lambda IAM policy related to the InvokeFunction action. I noticed you also fixed that issue already on main branch but not on the latest release 0.0.3. When are you planning to release a new version for this module?

For other users reference until we have a new release with that fix, change from:

https://github.com/aws-ia/terraform-aws-polygon-technology-edge/blob/v0.0.3/modules/security/data-sources.tf#L42

To:

```
"arn:aws:lambda:${var.region}:${var.account_id}:function:${var.lambda_function_name}"
```
